lib/locales/en.ts

```diff
@@ -980,6 +1005,7 @@ export const locale: Locale = {
     yolanda_of_poland_religious: 'Blessed Yolanda, Religious',
     zdenka_cecilia_schelingova_virgin: 'Blessed Zdenka Cecilia Schelingová, Virgin and Martyr',
     zdislava_of_lemberk: 'Saint Zdislava',
+    zechariah_and_elizabeth_parents_of_john_the_baptist: 'Saints Zechariah and Elizabeth, parents of John the Baptist',
```

Comment:
Same here as `twelve_apostles`, there is actually no matching calendar definitions.